src/tests/ftest/io/posix_stat.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2018-2021 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+from avocado.core.exceptions import TestFail
+
+from ior_test_base import IorTestBase
+from general_utils import get_remote_file_size, run_task
+from apricot import TestWithServers
+
+
+class POSIXStatTest(IorTestBase):
+    # pylint: disable=too-many-ancestors
+    """Test class description:
+
+    Requirement: SRS-10-0303
+    DFS stat call reports meaningful information (i.e. valid file size, correct
+    block size, ...).
+
+    Test Steps:
+    Create files of different size (block size) in a mounted POSIX container and
+    verify the size with stat command. Also verify the creation time with date
+    command.
+
+    :avocado: recursive
+    """
+    def test_stat_parameters(self):
+        """JIRA ID: DAOS-3769
+
+        Create files of 1M, 10M, 100M, 500M, and verify the size and creation
+        time.
+
+        :avocado: tags=all,full_regression
+        :avocado: tags=small
+        :avocado: tags=stat_parameters
+        """
+        block_sizes = self.params.get("block_sizes", "/run/*")
+        testfile_path_base = "/tmp/daos_dfuse/testfile"
```

Comment:
The `self.run_ior_with_pool()` method updates the `self.ior_cmd.test_file` property with the `self.dfuse.mount_dir.value` + `"testfile"` as part of it's execution, so this method does not need to define it:
```suggestion
```